spynl/main/mail.py

```diff
@@ -154,7 +154,7 @@ def send_template_email(request, recipient, template_string=None,
             **replacements)
 
     # Add a line to tell where the mail would be sent in production
-    if not is_production_environment() and mailer.__class__ is not DummyMailer:
+    if mailer.__class__ is not DummyMailer:
```

Comment:
This complete if statement and content of if statement needs to go. (This mail would in production etc...) The way it is now, this will be send to every email in the production environment.